teaching-platfrom/db_handler.py

```diff
@@ -0,0 +1,98 @@
+import yaml
+import psycopg2
```

Comment:
Using `psycopg2` directly it not encouraged. For this type of application `sqlalchemy` ORM will be sufficient, easier, safer and more readable.

Reply:
Done